styles/colors-gallium-day.less

```diff
@@ -149,10 +149,10 @@
 @agate-picker-active-color: inherit;
 @agate-picker-active-bg-color: @agate-accent;
 @agate-picker-active-bg-image: @agate-gradient;
-@agate-picker-item-top-bg-color: none;
```

Comment:
You may want to use "transparent" instead of "none" for background-color. "none" is not a valid value for the background-color property.